### ISSUE TITLE
TachiyomiEH abruptly closing when opening any manga from author search

### ISSUE THREAD

**Issue by kadaikhan** [NerdNumber9/TachiyomiEH#40]
App version: 7.4.4

Issue:
I recently change my device (now using galaxy note 8) and after installing and restoring my back up the app I tried to search an artist related works from my gallery but after selecting any work and trying to open any chapter it closes up the app.

I tried to search directly those titles by their name and it doesn't close when I open them up with that method.

**Comment by NerdNumber9** [comment]
Thanks for the report. I can't reproduce this on my device so I think some logs would help. It would be awesome if you could follow the steps below to get them:

Steps for taking logs:

1. Open TachiyomiEH and go through the steps you did to crash the app.
2. Immediately after the crash occurs, go to the `Settings` app on your phone, tap on `Developer options` and then tap on `Take bug report`. If you can't find the `Developer options` section, please refer to this article on how to enable it: https://www.digitaltrends.com/mobile/how-to-get-developer-options-on-android/
3. If asked about the type of bug report, choose `Interactive report`.
4. Open the notification shade and wait for the bug report to complete. Once finished, tap on the notification to send the bug report. You can email it to me at nerd.number9@yandex.com or upload it to Google Drive and post the link here.

Thanks!

**Comment by kadaikhan** [comment]
Thanks for replying.

After a few day of the problem I didn't use the app. I tried to replicate the issue but luckly, to my surprise, isn't happening anymore.

I do not know what was the thing that fixed it tho.

**Comment by ronalhvaldez** [comment]
I have the same problem. 

My bug report: https://drive.google.com/file/d/1ug_iQOTibg0fDWZylOalSANiBkaKvejR/view?usp=drivesdk

**Comment by bebebeben** [comment]
Same here. My issue seems to be caused by blocked sites. However, when I tried using a VPN it still crashed. This rarely happened before I updated to the latest version (7.4.4). Here's the crash log from Mi (I'm using Xiaomi 3S, couldn't find the one from the dev options so I figured I should just post them on the pastebin):

Without VPN
https://pastebin.com/w1Zp9WsX

With VPN
https://pastebin.com/VkPx80Uh

Would really appreciate it if you looked at them.

**Comment by eagle540** [comment]
Hello I'm new to this group but I was introduced to the app. Now I'm having the same issue we're soon as I open it it instantly crashes. I sent an email with my bug report. I hope to hear back from somebody soon. 

Update:
As I was messing with the app last night I discovered a couple things.

1) I was finally able to get the app not to crash as soon as I open it because I removed a bunch of H comic from my favorites. So maybe one issue was due to the amount I saved.
2) I noticed that all of manga that I saved from KissManga kept giving me an error message and would crash the app. So I remove those and for now it's working well.
3) when I went back to the gallery I noticed for some reason. No H comic we're appearing under the E-hentai gallery and when I went to KissManga no manga like bleach was showing and I got this exact error message in the app (SyntaxError: parse error (line1) ).

Hopefully this helps
